VaticanBAV/et/BAVet119.xml

```diff
@@ -0,0 +1,552 @@
+              <pubPlace>Hamburg</pubPlace>
+              <availability>
+                  <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                      licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+              </availability>
+          </publicationStmt>
+          <sourceDesc>
+              <msDesc xml:id="ms">
+                  <msIdentifier>
+                      <repository ref="INS0003BAV"/>
+                      <collection>Aethiopici</collection>
+                      <idno>Aeth.119</idno>
+                  </msIdentifier>
+                 
+                  <msContents>
+                      <summary/>
+                      <msItem xml:id="ms_i1">
+                          <locus from="3r" to="36v"/>
+                          <title type="complete" ref="LIT1171Awdana">ʿAwda nagaśt</title>
+                          <msItem xml:id="ms_i1.1">
+                              <locus target="#3r"/>
+                              <title type="complete">List of names and letters with which portions of the circle are annotated.</title>
+                          </msItem>
+                          <msItem xml:id="ms_i1.2">
+                              <locus target="#3v"/>
+                              <title>Figure consisting of a cross with four human faces at the ends 
+                                  of the four arms and an eye in the middle of each arm. 
+                                  Four circles, divided into four parts, fill the space between the arms.
+                                  Future events are indicated with this figure.</title>
+                          </msItem>
+                          <msItem xml:id="ms_i1.3">
+                              <locus from="4r" to="11v"/>
+                              <title type="complete" ref="LIT1171Awdana#tables">Sixteen circles, each divided into sixteen parts.</title>
+                              <incipit xml:lang="gez">
+                                  ፩ ዓውደ፡ ሆይ። ሀ ፩ትረክቦ፡ ለዜና፡ መፍቅድ፡ በባሕረ፡ ፃና።
+                              </incipit>
+                          </msItem>
+                          <msItem xml:id="ms_i1.4">
+                              <locus from="13r" to="36v" />
+                              <title type="complete" ref="LIT1171Awdana#seas">Sixteen chapters, indicated with the names of local lakes:
+                                  Ṣānā, Kʷarā, ʾElā, Šaklā, Zʷāy, ʾAlzazo, Wənǧ, <sic resp="MV">Goǧǧam</sic>, Dāgo, 
+                                  Ḥayq, Heno, Šemazbe, Ḥawaš, Gəb, Takaze, ʾAbāwi.</title>
+                              <incipit xml:lang="gez">
+                                  በስመ፡ አብ፡ <gap reason="ellipsis"/> ፩ አምላክ፡ መሐሪ፡ ወመስተሣህል፡ ዘተናገረ፡ ፩ ጠቢ<surplus>ቢ</surplus>ብ፡ እምነ፡ ብዙኃን፡ ጠቢባን፡ ወፈላስፋውያን፡
+                                  ዘንተ፡ ዘስሙ፡ መጽሐፈ፡ ምክር፡ ለአበው፡  <sic>ቀደመት፡</sic> ከመ፡ ያእምሩ፡ በቱ፡ በዝ፡ እምሂ፡ ሠናየ፡ ወዕመሂ፡ እኩየ፡
```

Comment:
በቱ፡ or ቦቱ፡ ?

Reply:
በቱ in the catalogue. I have marked it up with `<sic/>`